data/migrations/20181116114448_projects.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('projects', function (tbl) {
+    tbl.increments();
+    tbl.string('name', 255).notNullable
+    tbl.string('description')
+    tbl.boolean('completed')
```

Comment:
You can use .defaultTo(false) here and the field would populate false by default